### DISCUSSION TITLE
Custom preview and additional menu

### DISCUSSION THREAD

**Discussion by nucleartux** [enesozturk/react-native-hold-menu#64] [Ideas]
I wan't to do something like telegram menu or like this https://www.reddit.com/r/reactnative/comments/tz1pxh/how_would_you_create_such_a_blur_effect/ (I'm not an author).

**Comment by fukemy** [comment]
hi did you found any solution?

**Reply by nucleartux** [reply]
Yep, I'm using this library: https://github.com/dominicstop/react-native-ios-context-menu

**Reply by fukemy** [reply]
@nucleartux nu, thanks, I am using it too, it's really great library, I am finding the same library for android but still not find it

**Reply by nucleartux** [reply]
you could try https://zeego.dev/, it's wrapper around the library above on ios but they also provide some kind of solution for android

**Reply by fukemy** [reply]
@nucleartux thanks for you help, the main key is the `preview view`, zeego does not provide it's as I expected.
 `React-native-hole-menu` work for Android, but's it's render child 2 times when show the preview

**Reply by nucleartux** [reply]
oh, if you will find solution please let me know
